tests/fixtures/circuits/circuit_terminations.json

```diff
@@ -1,46 +1,3007 @@
 {
```

Comment:
Do we really need a 3000+ line fixture file? I'm in favor of updating *all* of the fixtures with up-to-date Nautobot data instead of the old quasi-NetBox data, but this seems like a lot more records here than our tests actually need.

Reply:
No we don't. This was just done quickly and copied from API docs call on demo.nautobot.com. I will go through records for each endpoint and make sure we cover the different types fields/data, but that should be handful at most.